support-frontend/assets/pages/supporter-plus-landing/twoStepPages/threeTierLanding.tsx

```diff
@@ -136,6 +136,28 @@ const tabletLineBreak = css`
 	}
 `;
 
+const suppportAnotherWayContainer = css`
+	margin: ${space[9]}px auto 0;
+	border-top: 1px solid ${palette.neutral[86]};
+	padding-top: 32px;
+	max-width: 940px;
+	text-align: left;
+	color: #606060;
+	h4 {
+		${textSans.medium({ fontWeight: 'bold' })};
+	}
+	p {
+		${textSans.small()};
+	}
+	a {
+		color: #606060;
+	}
+	${from.desktop} {
```

Comment:
Would this look better centered from tablet? Helen did'nt supply a Figma but maybe worth a check , or we do one big review at end which might be better?

Reply:
BIG REVIEW 👍